hack/test-cmd.sh

```diff
@@ -78,9 +84,15 @@ set -e
 
 function find_tests {
   find "${OS_ROOT}/test/cmd" -name '*.sh' -not -wholename '*images_tests.sh' | grep -E "${1}" | sort -u
+  exit 0
```

Comment:
No, why did you insert it? Best practice is to never `exit` in a function - always `return`